src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -747,6 +773,9 @@ def unit_ids(self) -> np.ndarray:
     def has_recording(self) -> bool:
         return self._recording is not None
 
+    def has_temporary_recording(self) -> bool:
+        return self._tmp_recording is not None
```

Comment:
should we make this function and variable have the same name?

so `has_tmp_recording` or `self._temporary_recording`?

I don't think this is crucial but might help us keep these straight...

Reply:
done